### DISCUSSION TITLE
Remove title section

### DISCUSSION THREAD

**Discussion by jimmisavage** [marcokreeft87/room-card#264] [General]
Hi,
I'm just wondering if it would be possible to remove the title section? Obviusly i could just remove the title but then i have the gap at the top of the card.

The reason i want to do this is because i'm using a markdown card to title a section and i then want to add a couple of room cards stacked on top of each other (but with a break between them).

**Comment by marcokreeft87** [comment]
Hey! No not really, that's because the card is designed to hold the entities of a room. But maybe with card_mod?

**Reply by jimmisavage** [reply]
Of course! I didn't even think of that :)

type: custom:room-card
style: |
  .title {
    min-height: 0px !important;
  }
